src/node_worker.cc

```diff
@@ -132,7 +132,16 @@ class WorkerThreadData {
  public:
   explicit WorkerThreadData(Worker* w)
     : w_(w) {
-    CHECK_EQ(uv_loop_init(&loop_), 0);
+    int ret = uv_loop_init(&loop_);
+    if (ret != 0) {
+      char err_buf[128];
+      uv_err_name_r(ret, err_buf, sizeof(err_buf));
+      std::string error_str = SPrintF("ERR_WORKER_INIT_FAILED: %s", err_buf);
+      w->custom_error_ = error_str;
```

Comment:
@HarshithaKP This is what’s problematic – `custom_err_` should only include the error id, i.e. `ERR_WORKER_INIT_FAILED`.

If you need to store additional information, that should probably be in a separate variable…

Reply:
@addaleax , Thanks. Made suggested changes to make it work.